core/src/main/scala/org/apache/spark/deploy/Client.scala

```diff
@@ -221,7 +221,9 @@ object Client {
     val conf = new SparkConf()
     val driverArgs = new ClientArguments(args)
 
-    conf.set("spark.rpc.askTimeout", "10")
+    if (!conf.contains("spark.rpc.askTimeout")) {
```

Comment:
can this ever be set? I don't remember whether the standalone master/worker reads config files or not.